sdk/resourcemanager/internal/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 0.3.1 (Unreleased)
+## 1.0.0 (Unreleased)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+* Export FakeCredential
 
 ### Other Changes
+* Upgrade to GA version of core lib and armresources
```

Comment:
Upgrade the module of azcore, azidentity and armresources to the stable version. 

Any module I missed?

Reply:
No, it's complete. I've changed the message.